content/en/project/partners/SHFT.md

```diff
@@ -0,0 +1,3 @@
+# Software History Futures and Technologies (SHFT) group
```

Comment:
Replace first line with the following `yaml`:
```
---
title: Software History Futures and Technologies (SHFT) group
weight: 10
type: docs
---
```

The `weight` value might need to be something else depending on where this should be positioned.